testsuite/forge-cli/src/main.rs

```diff
@@ -105,14 +116,25 @@ fn main() -> Result<()> {
                     &args.helm_repo,
                 )
             }
-            OperatorCommand::CleanUp(cleanup) => {
+            OperatorCommand::Resize(resize) => {
+                let _ = set_eks_nodegroup_size(
+                    args.cluster_name,
+                    resize.num_validators,
+                    resize.auth_with_k8s_env,
+                )
+                .unwrap();
```

Comment:
There's lots of places in this pr where you assign the return value of a function which is a `Result` to `_`. This ignores the error. Instead we probably want to propagate the error up like this:
 
```suggestion
                    set_eks_nodegroup_size(
                    args.cluster_name,
                    resize.num_validators,
                    resize.auth_with_k8s_env,
                )?;
```